README.md

```diff
@@ -31,14 +31,9 @@ DRIVER - Data for Road Incident Visualization, Evaluation, and Reporting
       (If you just want to install the DRIVER web interface, do this. You can add Android integration later.)
 
     - To build schema model JAR files for the Android app, copy the signing keystore to `gradle/data/driver.keystore`
-    and set the password for the keystore under `keystore_password` in `deployment/ansible/group_vars/all`.
+    and set the password for the keystore under `keystore_password` in `deployment/ansible/group_vars/development`.
```

Comment:
It might be helpful to add a note in the README suggesting `git update-index --assume-unchanged deployment/ansible/group_vars/development` to prevent changes to this file from being tracked by git. The reverse is `--no-assume-unchanged` and that'd be good to include also in case someone wants to update the default development settings.